# The joint state controller handles publishing transforms for any moving joints
franka_sim_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

franka_left_finger_controller:
  type: effort_controllers/JointPositionController
  joint: panda_finger_joint1
  pid: {p: 1000.0, i: 5.0, d: 100.0, i_clamp_min: -10.0, i_clamp_max: 10.0, antiwindup: true}

franka_right_finger_controller:
  type: effort_controllers/JointPositionController
  joint: panda_finger_joint2
  pid: {p: 1000.0, i: 5.0, d: 100.0, i_clamp_min: -10.0, i_clamp_max: 10.0, antiwindup: true}
